src/amo/pages/Collection/index.js

```diff
@@ -506,6 +506,22 @@ export class CollectionBase extends React.Component<InternalProps> {
     );
   }
 
+  getPageDescription() {
+    const { collection, i18n } = this.props;
+
+    invariant(collection, 'collection is required');
+
+    return i18n.sprintf(
+      i18n.gettext(`Firefox collections make it easy to keep track of favorite
+        extensions and themes, and share your perfectly customized browser with
+        others. Explore the %(collectionName)s: %(collectionDescription)s.`),
```

Comment:
it seems like description is optional so it looks a little odd if it's missing. See here:

![Alt text](https://monosnap.com/image/wG81jPjXMDm6SM7p0l7cvK9CEnvMkF.png)

can we show this without the `:` when there is no description available?

Also, just wondering on the wording here - would it sound better if it says "Explore the [CollectionName] Collection."  or "Explore [CollectionName]"? I bet this was already decided and covered somewhere but just double checking : )

Reply:
All Mozilla collections have a description (or should). Only Mozilla collections are indexed, so I guess it is fine to leave it as is for now.

Reply:
That being said, the latest revision makes it easier to handle both cases, so I did it :)

Reply:
seems fine to me : )